modules/yarn/manifests/init.pp

```diff
@@ -5,10 +5,10 @@
 # === Parameters:
 #
 # [*version*]
-#   Version to install. Defaults to 'present' which will install latest version
+#   Version to install. Defaults to '1.22.4-1'
 #
 class yarn(
-  $version = 'present',
+  $version = '1.22.4-1',
```

Comment:
I'm concerned that (like everything else), we won't remember to update this.

Reply:
Is there an alternative approach you'd recommend instead?

As it's a mirrored repo we already have to go into apt and publish a new snapshot to update Yarn. So it's already a rather manual process.